### PR TITLE
Simplified readme and added documentation about kinesis idea

### PR DESCRIPTION
**Pull request by humblelistener**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -18,49 +18,11 @@ This will help you have a head start with,
...
-3. Run CopyAWSCredentialsToUserProfile.sh in the root of the project dir
-4. Set a system variable for
-    - ENV = {your_username} (this will become part of your table/queue namespace in AWS in DC0)
-    - DC = DC0 (for local development)
-    - IsLocalServer = true (for local development so that ES hits this API)
-5. Restart VS for changes to take effect (if you get expiry errors while debugging restart visual studio again)
-6. Run the CommentsAPI project in debug
-
 ### What if I just want worker service?
 * Remove WebService & WebServer.UnitTests projects from Visual Studio and delete the folder
```
Comment (antonfelich) [inline Readme.md]: Should say 

> ... & WebService.UnitTests projects ...

Comment (humblelistener) [reply Readme.md]: fixed it, pull request updated!~

**Comment by antonfelich** [comment]
LGTM... can't work out how to approve it though :)

**Comment by humblelistener** [comment]
Have to say it is not straight forward. I haven't did one myself before,

Would this help, https://help.github.com/articles/approving-a-pull-request-with-required-reviews/

**Pull request merged**